mycore-ifs/src/main/java/org/mycore/datamodel/ifs/MCRContentStoreFactory.java

```diff
@@ -168,7 +167,7 @@ private static void initStoreSelector() {
 
         return EXTENDER_CLASSES.computeIfAbsent(storeID, key -> {
             String storeClass = "MCR.IFS.AVExtender." + key + CLASS_SUFFIX;
-            return MCRConfiguration.instance().getClass(storeClass, null);
+            return MCRConfiguration2.<org.mycore.datamodel.ifs.MCRAudioVideoExtender> getClass(storeClass).orElse(null);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.mycore.datamodel.ifs.MCRAudioVideoExtender' due to existing same package import 'org.mycore.datamodel.ifs.*'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5299284)